docs/modules/ROOT/pages/vshn-managed/postgresql/alerting.adoc

```diff
@@ -0,0 +1,61 @@
+= User Alerting
+
+You can enable alerting for your PostgreSQL instance either using an existing `alertmanagerConfig` or an Alertmanager Template. See below for configuration examples for both use cases.
+
+== From existing alertmanagerConfig
+
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNPostgreSQL
+metadata:
+  name: pgsql-app1-prod
+  namespace: prod-app
+spec:
+  parameters:
+    monitoring:
+      alertmanagerConfigSecretRef: alert-secret # <1>
+      alertmanagerConfigRef: my-alert # <2>
+    service:
+      majorVersion: "15"
+  writeConnectionSecretToRef:
+    name: postgres-creds
+----
+<1>: Name of the secret in the current namespace
```

Comment:
This is the only reference to the secret. I'm not sure if this is understandable by someone the doesn't already understand how it works.

Reply:
I tried to improve a bit the overall page. Can you check again?